tests/OnlineSales.Tests/BaseTest.cs

```diff
@@ -84,6 +85,26 @@ protected Task<HttpResponseMessage> Request(HttpMethod method, string url, objec
         return Client.SendAsync(request);
     }
 
+    protected Task<HttpResponseMessage> Request(HttpMethod method, string url, TestImage? payload, string authToken = "Success")
```

Comment:
This method should be moved away from BaseClass. Please put it to ImagesTests.cs as it it very specific to that particular test class. We do not need it in any other test cases.